renpy/ast.py

```diff
@@ -2201,9 +2201,22 @@ class UserStatement(Node):
     translation_relevant: bool = False # type: ignore
     rollback: RollbackType = "normal"
     subparses: list['renpy.lexer.SubParse'] = [ ]
-    init_priority: SignedInt | None = 0
     atl: 'renpy.atl.RawBlock | None' = None
 
+    init_priority: SignedInt | None = None
```

Comment:
Redefining this default will cause any node parsed prior to this change with a value of `0` to now have a value of `None`.